.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
     "**/*.js": { "when": "$(basename).ts" },
     "**/*.d.ts": { "when": "$(basename).ts" },
     "**/*.js.map": true
-  }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This defaults VSCode to using the local version of TS instead of the bundled version.